.github/workflows/triage_issues.yml

```
@@ -12,6 +12,7 @@ jobs:
     runs-on: ubuntu-slim
     steps:
       - name: "Add label to issue"
+        if: ${{ secrets.ADD_TO_PROJECT_PAT != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)